core/src/main/java/io/grpc/internal/ServerCallImpl.java

```diff
@@ -121,33 +123,45 @@ public boolean isCancelled() {
     return cancelled;
   }
 
-  ServerStreamListenerImpl newServerStreamListener(ServerCall.Listener<ReqT> listener,
+  ServerStreamListenerImpl<ReqT> newServerStreamListener(ServerCall.Listener<ReqT> listener,
```

Comment:
I only see newServerStreamListener used in one place, and its treated as a ServerStreamListener.  Since ServerStreamListenerImpl was private until now, no one could have known if it was generic or not.  

I can make newServerStreamListener to keep the parameter from escaping.